tasks/smoketests/tasks.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os.path
+from invoke import task
+from .vscode.application import get_options, install_extension
+from .vscode.download import download_chrome_driver as download_chrome_drv
+from .vscode.download import download_vscode
+from .bootstrap.main import build_extension
+from .main import start_smoketests
+
+
+@task(name="download")
+def download_all(ctx, destination=".vscode-smoke", channel="stable"):
+    """Downloads VS Code (stable/insiders) and chrome driver.
+
+    The channel defines the channel for VSC (stable or insiders).
+    """
+    destination = os.path.join(destination, "vscode")
+    download_vscode(destination, channel)
+    download_chrome_drv(destination, channel)
+
+
+@task(name="download_vsc")
+def download_vsc(ctx, destination=".vscode-smoke", channel="stable"):
+    """Downloads VS Code (stable/insiders)."""
```

Comment:
```suggestion
    """Download VS Code (stable/insiders)."""
```

I'm going to stop changing the tense as I think you get the idea. :)